x/concentrated-liquidity/internal/model/pool.go

```diff
@@ -15,20 +15,24 @@ var (
 	_ types.ConcentratedPoolExtension = &Pool{}
 )
 
-func NewConcentratedLiquidityPool(poolId uint64, denom0, denom1 string, currSqrtPrice sdk.Dec, currTick sdk.Int) (Pool, error) {
+func NewConcentratedLiquidityPool(poolId uint64, denom0, denom1 string, currSqrtPrice sdk.Dec, tickSpacing uint64) (Pool, error) {
+	// Order the initial pool denoms so that denom0 is lexicographically smaller than denom1.
 	denom0, denom1, err := types.OrderInitialPoolDenoms(denom0, denom1)
 	if err != nil {
 		return Pool{}, err
 	}
+
+	// Create a new pool struct with the specified parameters
 	pool := Pool{
 		// TODO: move gammtypes.NewPoolAddress(poolId) to swaproutertypes
 		Address:          gammtypes.NewPoolAddress(poolId).String(),
 		Id:               poolId,
-		CurrentSqrtPrice: currSqrtPrice,
-		CurrentTick:      currTick,
+		CurrentSqrtPrice: sdk.ZeroDec(),
+		CurrentTick:      sdk.ZeroInt(),
```

Comment:
Does it make sense t initialize the current tick here? Zero is a valid value for a tick. To signify, that it is uninitialized, maybe we should keep it as nil int. Wdyt?

Reply:
Discussed this offline, going to keep this at zero for now, since sdk numbers when marshaled will always default to zero if nil.